src/components/HomePage/Banner.js

```diff
@@ -81,41 +97,42 @@ const useStyles = makeStyles(theme => ({
 
 function renderInfo(classes) {
 	const tz = getTimeZoneWithFormat(hothStart, 'short');
-	const month = monthFormatter.format(hothStart);
+	//const month = monthFormatter.format(hothStart);
 	const startDay = hothStart.getDate();
 	const endDay = hothEnd.getDate();
 	const eventCrossesDate = startDay !== endDay;
-	const endDayString = eventCrossesDate ? `–${endDay}` : '';
+	//const endDayString = eventCrossesDate ? `–${endDay}` : '';
 	return (
 		<Grid
 			container
 			direction='column'
-			justify='center'
-			alignItems='center'
+			justify='left'
+			alignItems='left'
 		>
-			<Typography variant='h3' className={classes.text} component='h2' style={{ fontSize: '3em' }}>
-				Hack off the Hill 8
+			<img src = {hothTitle}
+			style={{ paddingLeft: 50, paddingBottom: 20, width: '80%' }}
+			/>
+
+			<Typography variant='h5' className={classes.text} component='h3'
+			style={{ paddingLeft: 57, fontWeight: 500 }}>Explore. Build. Empower.
 			</Typography>
-			<Typography variant='h5' className={classes.text} component='h3'>Explore. Build. Empower.</Typography>
-			<SvgImg src={hothLogo} className={classes.logo} width={258} height={250}
-				style={{
-					width: 150,
-					margin: 20
-				}}/>
-			<Box display='flex' alignItems='center' color='white'>
+
+			<Box display='flex' alignItems='left' color='white'>
 				<Tooltip title={eventCrossesDate ?
 					`It looks like HOTH crosses between dates in your timezone! (${tz})` :
 					''}
 				placement='top'
 				arrow={true}>
-					<Typography variant='h5' className={classes.text} component='h3'>
-						<time dateTime={hothStart.toISOString()}>
-							{month} {startDay}{endDayString}, 2021
-						</time>
+					<Typography variant='h5' className={classes.text} style={{ paddingLeft: 57, paddingTop: 40 }} component='h3'>
+							February 5 - 7, 2021 | Virtual
```

Comment:
We used variables here because the start/end date may differ for people in other time zones.

Also, we should keep the [`time` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) here since that provides more information for screen readers.